### ISSUE TITLE
todo

### ISSUE THREAD

**Issue by rosikand** [rosikand/ddpm-pytorch#1]
- try with hugging face u-net 
- try the artbench or cifar data with the HF pipeline to see how well that data can be learned and with how many samples needed.

**Comment by rosikand** [comment]
Debugging: 

 - [x] [normalize](https://discuss.pytorch.org/t/how-to-normalize-image-data-from-1-to-1/145502) to [-1, 1] instead of /255. 
 - gradient accumulation


note: forward diffusion working... cross-tested with lucidrains implementation.

**Comment by rosikand** [comment]
- [x] add [more variance scheduler](https://huggingface.co/blog/annotated-diffusion#defining-the-forward-diffusion-process) options:

**Comment by rosikand** [comment]
things to try:
- use the same noise vector sample for each sample in the batch
